www/source/partials/docs/_dev-pkgs-exports.html.md.erb

```diff
@@ -140,6 +140,63 @@ The Kubernetes exporter is an additional command line subcommand to the standard
     ```shell
     $ hab pkg export kubernetes --push-image --username <your_docker_hub_username> --password <your_docker_hub_password> -o my_app.yaml
     ```
+6. Please note that you will need to add the HAB_LICENSE environment variable to the generated manifest yaml file. One way to do this is to add the environmental variable directly to the generated manifest like this example:
+
+    ```yaml
+    ---
+    apiVersion: habitat.sh/v1beta1
+    kind: Habitat
+    customVersion: v1beta2
+    metadata:
+    ## Name of the Kubernetes resource.
+    name: sample-node-app-1-1-0-20190516204636
+    spec:
+    v1beta2:
+        ## Name of the Habitat service package exported as a Docker image.
+        image: nellshamrell/sample-node-app:1.1.0-20190516204636
+        ## Number of desired instances.
+        count: 1
+        ## An object containing parameters that affects how the Habitat service
+        ## is executed.
+        service:
+        ## Name of the Habitat service.
+        name: sample-node-app
+        ## Habitat topology of the service.
+        topology: standalone
+        env:
+        - name: HAB_LICENSE
+        value: accept-no-persist
```

Comment:
Does the indentation level of `value` need to match that of `name`?  As it is now, it looks like `value` is a peer of `env`, which doesn't seem right.

Reply:
That was likely copy pasta - fixing!

Reply:
Fixed!